windowslib/src/main/scala/scala/scalanative/windows/util/HelperMethods.scala

```diff
@@ -0,0 +1,28 @@
+package scala.scalanative.windows.util
+
+import scala.scalanative.unsafe._
+import scala.scalanative.windows.HandleApi.Handle
+import scala.scalanative.windows.ProcessThreadsApi._
+import scala.scalanative.windows._
+
+object HelperMethods {
+  def withUserToken[T](desiredAccess: DWord)(fn: Handle => T): T = {
```

Comment:
Shouldn't this be private to some scope?

Reply:
Those helper methods are mostly used in javalib. Should I move it there and made it `private[java]`?

Reply:
Yes, please.